docs/content/concepts/dagster-pipes/pyspark.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: "Integrating PySpark with Dagster Pipes | Dagster Docs"
+description: "Learn to integrate Dagster Pipes with PySpark to orchestrate PySpark jobs in a Dagster pipeline."
+---
+
+# PySpark & Dagster Pipes
+
+This tutorial is focused on using Dagster Pipes to launch & monitor general PySpark jobs. The [Spark integration page](/integrations/spark) provides more information on using Pipes with specific Spark provides (such as AWS EMR or Databricks).
+
+Spark is often used with object stores such as Amazon S3. We are going to simulate this setup locally with [minio](https://min.io/) which has an API compatible with AWS S3. All communication between Dagster and the Spark job (metadata and logs collection) will happen through a Minio bucket.
+
+---
+
+## Prerequisites
+
+We are going to use `docker` to emulate a typical Spark setup locally. Therefore, only `docker` is required to reproduce this example.
```

Comment:
I would format this as just "Docker" and link to the Docker installation page: https://docs.docker.com/get-started/get-docker/